src/guide/advanced/isi.md

```diff
@@ -1,77 +1,29 @@
-Instructions (ISI)_, and there is no other way of modifying the world
-state. To understand why, we'll need to make a short detour into how Iroha
-is implemented under the hood.
-
-## How Iroha works
-
-Each interaction with the blockchain is done via a _transaction_. A
-transaction is a collection of _instructions_, which are either glued
-together in sequence or compiled into what we affectionately call a WASM
-blob (more on that later). You need these instructions to register an
-account, remove an account, add X amount of Y currency, and so on.
-
-To read the information encoded in the blocks of a blockchain (the current
-world state), you use _queries_. Queries are submitted like instructions,
-but they're not tracked and recorded in blocks, so they're much more
-lightweight. If you use queries as part of complicated logic (e.g. inside
-WASM), they have a non-negligible impact on the size of the blocks. Queries
-that are only used to get information leave no trace in the blockchain.
-
-### Consensus
-
-Each time you send a transaction to Iroha, it gets put into a queue. When
-it's time to produce a new block, the queue is emptied, and the consensus
-process begins. This process is equal parts common sense and black
-magic[^1].
-
-The mundane aspect is that a special set of peers needs to take the
-transaction queue and reproduce the same world state. If the world state
-cannot be reproduced for some reason or another, none of the transactions
-get committed to a block.
-
-The consensus starts over from scratch by choosing a different special set
-of peers. This is where the black magic comes in. There is a number of
-things that are fine-tuned: the number of peers in the voting process, the
-way in which subsequent voting peers are chosen, and the way in which the
-peers communicate that consensus has failed. Because this changes the view
-of the world, the process is called a _view change_. The exact reason for
-why the view was changed is encoded in the _view change proof_, but
-decoding that information is an advanced topic that we won't cover here.
-
-The reasoning behind this algorithm is simple: if someone had some evil
-peers and connected them to the existing network, if they tried to fake
-data, some good™ peers would not get the same (evil™) world state. If
-that's the case, the evil™ peers would not be allowed to participate in
-consensus, and you would eventually produce a block using only good™ peers.
-
-As a natural consequence, if any changes to the world state are made
-without the use of ISI, the good™ peers cannot know of them. They won't be
-able to reproduce the hash of the world state, and thus consensus will
-fail. Same thing happens if the peers have different instructions.
-
-## Instructions
-
-So what kinds of special instructions do we have? If you've read the
+When we talked about [how Iorha operates](intro.md#how-iroha-works), we
+said that Iroha Special Instructions is the only way to modify the world
```

Comment:
```suggestion
said that Iroha Special Instructions are the only way to modify the world
```

Reply:
isn't it similar to this case?
<img width="752" alt="image" src="https://user-images.githubusercontent.com/17865025/190360478-625b66b1-2d25-42a4-8499-35d44e47f8ee.png">
https://webapps.towson.edu/ows/sub-verb.htm

Reply:
Not really, the instructions are plural, which is why I used the **are**. If it was instruction (singular) then **is** would be more accurate